src/components/VideoItem.tsx

```diff
@@ -9,16 +9,16 @@ const VideoItem = ({
   handleVideoSelect: (video: YTVideo) => any;
 }): any => {
   return (
-    <div onClick={() => handleVideoSelect(video)} className=" video-item item">
-      <img
-        className="ui image"
-        src={video.snippet.thumbnails.medium.url}
-        alt={video.snippet.description}
-      />
-      <div className="content">
-        <div className="header ">{video.snippet.title}</div>
+    <React.Fragment>
```

Comment:
Did not know about Fragments! Very useful component, thanks!

Reply:
Ya great thing to pile up stuff.